lms/templates/courseware/proctored-exam-status.underscore

```diff
@@ -0,0 +1,19 @@
+<div class="exam-timer">
+    <%- gettext("You are taking \"") %>
```

Comment:
My comments regarding this file got collapsed because a PR was merged with text changes. You can go back and look over them, but in summary:

1) You can't do string concatenation like this (screws up translation)
2) Make sure to escape the exam_display_name (I think you can use <%- instead of <%=)
3) !== instead of !=